index.js

```
@@ -186,6 +186,65 @@ app.delete("/habits/:habitId", authenticate, (req, res) => {
   checkHabitStmt.finalize();
 });
 
+app.get("/habits/:habitId/trackers", authenticate, (req, res) => {
+  const habitId = req.params.habitId;
+  const userId = req.userId;
+  const startDate = req.query.startDate;
+  const endDate = req.query.endDate;
+
+  if (
+    (startDate && isNaN(new Date(startDate).getTime())) ||
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)